src/net/client/stream.rs

```diff
@@ -852,23 +862,25 @@ mod test {
         let mut queries = Queries::new();
 
         for i in 0..12 {
-            let (idx, item) = queries.insert(i).unwrap();
+            let (idx, item) = queries.insert(i).expect("test failed");
```

Comment:
Are these `expect()` calls really any more useful than the `unwrap()` calls? In both cases you will have to look at the code to find out where it failed as the expect messages are not distinct thus don't tell you where it failed, and the `unwrap()` calls make the code shorter.

Reply:
The main point is that it is easy to verify that there are no unwraps in production code. If there is another way to grep for unwrap that excludes unit tests, then let me know.

Reply:
I don't agree that support for grepping for `unwrap` should be a goal. As it is test code, leave it.